pom.xml

```diff
@@ -36,6 +36,7 @@
         <module>elasticjob-lite</module>
         <module>elasticjob-cloud</module>
         <module>elasticjob-distribution</module>
+        <module>elasticjob-error-handler</module>
```

Comment:
This module maybe put a better sequence, before elasticjob-lite is a better place